bin/yii

```diff
@@ -37,7 +37,10 @@ use Yiisoft\Yii\Console\Application;
     }
     require_once $autoloadPath;
 
-    $container = new Container(require Builder::path('console'));
+    $params = require Builder::path('params');
+
+    /** @var ContainerInterface $container */
+    $container = $params['container']();
```

Comment:
How about to use full class name as a key?

Reply:
Or `console.container`

Reply:
`console.container` sounds good.